hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -106,13 +109,13 @@ public void preHandle(
      */
     public void handle(@NonNull final TransactionMetadata metadata) {
         // TODO : Need to implement this method when we are ready to validate payments for query
-        //        requireNonNull(metadata);
-        //        throw new UnsupportedOperationException("Not implemented");
+        requireNonNull(metadata);
+        throw new UnsupportedOperationException("Not implemented");
     }
 
     private void handleTokenTransfers(
             final List<AccountAmount> transfers, final PreHandleContext meta, final ReadableAccountStore accountStore) {
-        for (AccountAmount accountAmount : transfers) {
+        for (final AccountAmount accountAmount : transfers) {
```

Comment:
null check for transfers

Reply:
I don't think they can be null. It will have a default list